### PR TITLE
[kotlin-client][kotlin-server] Feature/kotlin data class serial version uid

### PR DESCRIPTION
**Pull request by janweinschenker**
…rializable, by adding the config option serializableModel=(true|false)

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language. @jimschubert (2017/09) ❤️, @dr4ke616 (2018/08) @karismann (2019/03) @Zomzog (2019/04)

### Description of the PR

If kotlin data classes implement the interface java.io.Serializable, they should have a companion object defining a serialVersionUID of type Long.

Therefore, these templates have been changed accordingly:

* kotlin-server/data_class.mustache
* kotlin-client/data_class.mustache

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache [hunk 1]
```diff
@@ -1,69 +1,67 @@
...
-import kotlinx.android.parcel.Parcelize
+	{{#gson}}
+		import com.google.gson.annotations.SerializedName
+	{{/gson}}
+	{{#moshi}}
+		import com.squareup.moshi.Json
+	{{/moshi}}
+	{{#parcelizeModels}}
+		import android.os.Parcelable
+		import kotlinx.android.parcel.Parcelize
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache]: @janweinschenker did you use an IDE to reformat the mustache templates?

Comment (janweinschenker) [reply modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache]: yes, sorry. Reformatting was not intended. I could revert to the original formatting. Should I?

Comment (wing328) [reply modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache]: No need to sorry. Things happened.

Please undo the formatting changes to the templates

Comment (janweinschenker) [reply modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache]: will do

Comment (janweinschenker) [reply modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache]: it's done

**Comment by wing328** [comment]
Please run the following to update Kotlin client samples:
```
./bin/kotlin-client-petstore-multiplatform.sh
./bin/kotlin-client-petstore.sh
./bin/kotlin-client-string.sh
./bin/kotlin-client-threetenbp.sh
```

**Comment by wing328** [comment]
CircleCI caused by outdated doc. I'll fix that with another PR.

**Comment by wing328** [comment]
@janweinschenker thanks for the PR, which has been included in v4.2.0 release: https://twitter.com/oas_generator/status/1189824932345069569

**Pull request merged**